### PR TITLE
Fix a build failure for `sbt clean +publishLocal`

### PR DESCRIPTION
**Pull request by plokhotnyuk**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -64,3 +64,13 @@ buck-out/
 
 ########### Bundle artifact ###########
 *.jsbundle
+
+########### IntelliJ IDEA ###########
```
Comment (sirthias) [inline .gitignore]: These things shouldn't be in the project-level `.gitignore`, but in your personal one that's global to your dev machine. (Also see the comment at the top of this file.)

File: core/src/main/scala/io/bullet/borer/json/JsonParser.scala [hunk 2]
```diff
@@ -20,7 +20,7 @@ import scala.annotation.{switch, tailrec}
   * Encapsulates the basic JSON parsing logic.
   * Also performs inline UTF-8 decoding from raw bytes.
   *
-  * This [[Receiver.Parser]] only produces data items that can be directly represented in JSON, specifically
+  * This [[Receiver]] only produces data items that can be directly represented in JSON, specifically
```
Comment (sirthias) [inline core/src/main/scala/io/bullet/borer/json/JsonParser.scala]: ok, yes, thank you!

File: .gitignore (hunk 1 above)
Comment (plokhotnyuk) [reply .gitignore]: Sorry... I have removed unwanted commit and rebased remaining one.

**Comment by sirthias** [comment]
Great. Thank you once again, @plokhotnyuk !

**Pull request merged**